### DISCUSSION TITLE
gastby-source-drupal - still use deprecated ___NODE

### DISCUSSION THREAD

**Discussion by kmajzlik** [gatsbyjs/gatsby#34097] [Help]
Please how exactly change this code?
I have Node of type Movie. This has field_image = relation to Media of type image. This has field_media_image which is file--file.

I still use old gatsby-image. I tried many ways but either gatsby fail or i lost all images on frontend.

```
exports.createSchemaCustomization = ({ actions }) => {
  const {createTypes} = actions
  const typeDefs = `
    type media__imageRelationships {
      field_media_image: file__file @link(from: "field_media_image___NODE"),
    }
    type media__image implements Node {
      relationships: media__imageRelationships
    }

    type node__movieRelationships {
      field_image: file__file @link(from: "field_image___NODE"),
    }
    type node__movie implements Node {
      relationships: node__movieRelationships
    }
  `;
  createTypes(typeDefs);
}
```

**Comment by kmajzlik** [comment]
Ok, i see problem is directly in gatsby-source-drupal on several places.
This should be fixed there.
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-drupal/src/utils.js#L57
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-drupal/src/utils.js#L115
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-drupal/src/utils.js#L377
and
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-drupal/src/normalize.js#L128

**Comment by laneparton** [comment]
This has also been unanswered here: https://github.com/gatsbyjs/gatsby/discussions/35259
